homeassistant/components/deconz/sensor.py

```diff
@@ -305,12 +324,13 @@ def unique_id(self):
         return f"{self.serial}-battery"
 
     @property
-    def native_value(self):
+    def native_value(self) -> int:
         """Return the state of the battery."""
+        assert isinstance(self._device.battery, int)
```

Comment:
We should not have to assert the instance here

Reply:
Fixed